### DISCUSSION TITLE
Localize AuthenticationException message

### DISCUSSION THREAD

**Discussion by cirolosapio** [laravel/framework#53321] [General]
in the [Authenticate](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Auth/Middleware/Authenticate.php#L103) middleware we have 'Unauthenticated.' message exception without ability to customize for localization

**Comment by mbabker** [comment]
By default, exception messages should be treated as informational messages for developers and not text that is arbitrarily shown to any end user.  So the middleware doesn't need to support translating this message out-of-the-box IMO.

There are two solutions if you need these messages translated:

1) In the case of this middleware specifically, add a middleware in your application extending the framework class and replace the `unauthenticated()` method with your own implementation that supports translations

2) As a general practice, you can add some logic in your application's exception handler to take care of creating translated messages when necessary
